client/landing/top-links.tsx

```diff
@@ -78,42 +99,84 @@ const HideWhenSmall = styled.span`
   }
 `
 
-const TopLinks = () => {
+export function TopLinks({ className }: { className?: string }) {
+  const { i18n } = useTranslation()
+  const dispatch = useAppDispatch()
+  const [languageMenuOpen, openLanguageMenu, closeLanguageMenu] = usePopoverController()
+  const [anchor, anchorX, anchorY] = useAnchorPosition('left', 'bottom')
+
+  const onChangeLanguage = useStableCallback(async (language: TranslationLanguage) => {
+    closeLanguageMenu()
+    try {
+      await i18n.changeLanguage(language)
+    } catch (error) {
+      dispatch(openSnackbar({ message: 'Something went wrong changing the language' }))
```

Comment:
This should be a translated string.

Also probably: "Something went wrong when changing the language"

Reply:
(If you have another PR in the works for translating the rest of the content here, it's fine to leave the translation until then, but probably good to get in the habit of translating everything from here on out)